common/src/main/resources/index-mappings/ml_memory_container.json

```
@@ -0,0 +1,54 @@
+{
+  "_meta": {
+    "schema_version": 1
+  },
+  "properties": {
+    "container_id": {
+      "type": "keyword"
+    },
+    "container_name": {
```

Suggest follow current naming convention of model/agent, just use `name`
```suggestion
    "name": {
```